src/trans-word-level/property.cpp

```diff
@@ -663,9 +678,19 @@ static obligationst property_obligations_rec(
     auto &sequence =
       to_sva_sequence_property_expr_base(property_expr).sequence();
 
+    // clang-format off
+    auto semantics =
+        property_expr.id() == ID_sva_strong ? sva_sequence_semanticst::STRONG
+      : property_expr.id() == ID_sva_weak ? sva_sequence_semanticst::WEAK
+      : property_expr.id() == ID_sva_implicit_strong ? sva_sequence_semanticst::STRONG
+      : property_expr.id() == ID_sva_implicit_weak ? sva_sequence_semanticst::WEAK
+      : sva_sequence_semanticst::WEAK;
+    // clang-format on
```

Comment:
That's the same code (except different expression variable) twice in the same file - should probably be moved to a (`static`) function.